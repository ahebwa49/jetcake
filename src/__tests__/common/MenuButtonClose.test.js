import React from "react";
import configureStore from "redux-mock-store";
import { shallow, mount } from "enzyme";
import { Provider } from "react-redux";
import toJson from "enzyme-to-json";

import MenuButtonClose from "../../common/MenuButtonClose";

const mockStore = configureStore([]);

let component;

component = shallow(<MenuButtonClose />);

test("MenuButtonClose component renders without crashing", () => {
  expect(toJson(component)).toMatchSnapshot();
});
