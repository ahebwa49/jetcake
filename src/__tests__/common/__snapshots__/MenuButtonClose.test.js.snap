// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MenuButtonClose component renders without crashing 1`] = `
<div
  style={
    Object {
      "alignItems": "center",
      "color": "white",
      "cursor": "pointer",
      "display": "flex",
      "flexDirection": "column",
      "height": "35px",
      "justifyContent": "center",
      "width": "35px",
    }
  }
>
  <div
    style={
      Object {
        "backgroundColor": "black",
        "borderRadius": "3px",
        "height": "5px",
        "transform": "rotate(45deg)",
        "width": "100%",
      }
    }
  />
  <div
    style={
      Object {
        "backgroundColor": "black",
        "borderRadius": "3px",
        "height": "5px",
        "position": "relative",
        "top": "-5px",
        "transform": "rotate(-45deg)",
        "width": "100%",
      }
    }
  />
</div>
`;
